### DISCUSSION TITLE
Log user actions

### DISCUSSION THREAD

**Discussion by enrico-ghidoni** [CntoDev/level-up#131] [Ideas]
It would be nice to log actions from Interviewers and Member Managers (i.e. accepting Recruit Application Form, marking bootcamp as done) that can be checked out in a dedicated section.

**Comment by milivojm** [comment]
I thought I've done that one, check out `EventStates` table. It logs each event before sending to event bus.

It's not visible though.

**Comment by enrico-ghidoni** [comment]
OK then it's only a UI thing
